freegames/typing.py

```diff
@@ -0,0 +1,96 @@
+""" Typing, typing game to remove the same letters.
+
+Exercises
+
+1. Change the speed of letters.
+2. Change letters to simple words.
+3. Add bonus words.
+
+"""
+from random import *
+from turtle import *
+from freegames import vector
```

Comment:
Add `from string import ascii_lowercase as letters` and then use `letters` throughout the program instead of `string.ascii_lowercase` as I've done.